### ISSUE TITLE
Create Canonical Permissions Lists for Equipment

### ISSUE THREAD

**Issue by DoESsean** [DoESLiverpool/somebody-should#1237]
Since we now also have the CNC which requires induction as well as the lasers, @amcewen and I were discussing the notion of having a list (probably online for the sake of ease) that will be our canonical reference for who can use what equipment in the space. This would also tie into the idea of having a 'traffic light' system for the safety of each item, with different levels of permission being required to certify on the different equipments.

So, for example, only an Organiser or Inductor would be able to certify someone to be allowed to use the lasers or CNC, but anyone could self certify for, for example, the 3D printer.

**Comment by Sean-anotherone** [comment]
I recall suggesting something similar last year....
Definitely support this, together with a file of safety info adjacent to each piece of kit.  This is the simplest way to manage multiple users.

**Comment by DoESsean** [comment]
Right, it seems like the sensible place to have this would be on the Wiki, that way everyone can access it and see who actually has permissions to use the CNC and lasers. However, I'm not sure if that would mean it would be editable by anyone, which would defeat the object.

For other equipment, I think a simple sign in sheet would suffice, as we don't specifically need people to be inducted, rather it's self-certification. This would, however, let us know that people have said they know what they are doing and would (I think?) give us some protections if they are hurt while using them. If we create the safety files as suggested by @Sean-anotherone then these sheets could be kept along with them by each piece of equipment.

@Sean-anotherone, what wuld these files need to include? Take the 3d printers, for example - would it need to be as complex as including the instructions for use, model information, etc, or as simple as something like, 'Warning - nozzle and plate get extremely hot. Do not touch whilst print is in progress'?

**Comment by arthurrowland** [comment]
There was a CNC Team induction sheet at some point but I can't see it in the drawer. Does anyone know where that went? Be useful to feed that into the list discussed here.
